controllers/heat_controller.go

```diff
@@ -568,6 +568,17 @@ func (r *HeatReconciler) reconcileNormal(ctx context.Context, instance *heatv1be
 		return ctrlResult, err
 	}
 
+	// remove finalizers from previous MariaDBAccounts for which we have
+	// switched.
+	// TODO(zzzeek) - This is here because I dont have test fixtures to
```

Comment:
this comment refers to when is it safe to delete finalizers from old mariadbaccounts, meaning, are we certain that all pods are re-deployed with a new username/password by the time we get here